test/lit.cfg.py

```diff
@@ -98,3 +98,34 @@ def set_system_desc_features(system_desc):
     ],
     append_path=True,
 )
+
+# add test related parameters
+test_arch = os.getenv("TEST_ARCH", "").split(",")
+test_pipeline = os.getenv("TEST_PIPELINE", "").split(",")
+test_target_family = os.getenv("TEST_TARGET_FAMILY", "").split(",")
+test_target_silicon = os.getenv("TEST_TARGET_SILICON", "").split(",")
+test_duration = os.getenv("TEST_DURATION", "").split(",")
```

Comment:
Hey @tapspatel, I think arch + family + silicon needs to be driven by system desc which is already wired up.  See line 25 & 59 of this file.